public/app/features/explore/Wrapper.tsx

```diff
@@ -88,19 +88,29 @@ class WrapperUnconnected extends PureComponent<Props> {
   }
 
   render() {
-    const { left, right } = this.props.queryParams;
+    const { left, right, search } = this.props.queryParams;
```

Comment:
This is too hacky because it relies on NavBar component behavior which may change (e.g. `search` is renamed or removed) and Explore should not depend on it.

It looks more like a problem with z-indexing of DashboadSearch vs ExploreDrawer. `DashboardSearch` has z-index of 1001 while ExploreDrawer is 1060. What is more we will not have the `DashboardSearch` once `topnav` is enabled.

Reply:
I realize it was a hacky solution, I thought it may be okay because topnav will make this no longer an issue, but I have changed the solution to use z-index.